compiler/plugins/target/ROCM/builtins/tuning/iree_default_tuning_spec_gfx942.mlir

```diff
@@ -19,6 +19,77 @@ transform.named_sequence @apply_op_config(%op: !transform.any_op {transform.read
   transform.yield
 }
 
+transform.named_sequence @apply_attn_op_config(%attention: !transform.any_op {transform.readonly},
+                                                %config: !transform.any_param {transform.readonly},
+                                                %decomposition_config: !transform.any_param {transform.readonly}) {
+  transform.annotate %attention "compilation_info" = %config : !transform.any_op, !transform.any_param
+  transform.annotate %attention "decomposition_config" = %decomposition_config : !transform.any_op, !transform.any_param
+  transform.annotate %attention "__tuning_spec_applied__" : !transform.any_op
+  transform.yield
+}
```

Comment:
Can you add a comment here `//* Tuning Configurations Start *//` or something like that seprating utilities and tuning specs.